skilloracle/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+# This file is a WIP
```

Comment:
If this is to tie all the dockerfiles together, should it be in the top level instead of the skill-oracle directory?

Reply:
Agreed, although the oracle service does depend on the official docker `redis` image we'll have to figure out a good way to express that dependency w/in the skill oracle service